test/hash_code_test.dart

```diff
@@ -55,9 +55,9 @@ void main() {
 
   test('testHashCodeOptionalEnum', () {
```

Comment:
I tend to like separating dartfmt commits from everything else.

Makes it easier to understand what happens in a given patch.